wire-ios-data-model/Tests/MLS/SafeCoreCryptoTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  SafeCoreCryptoTests.swift
+//  WireDataModelTests
+//
+//  Created by F on 22/02/2023.
+//  Copyright © 2023 Wire Swiss GmbH. All rights reserved.
+//
+
+import Foundation
+@testable import WireDataModel
+
+func createTempFolder() -> URL {
+    let url = URL(fileURLWithPath: [NSTemporaryDirectory(), UUID().uuidString].joined(separator: "/"))
+    try! FileManager.default.createDirectory(at: url, withIntermediateDirectories: true, attributes: [:])
+    return url
+}
+
+class SafeCoreCryptoTests: ZMBaseManagedObjectTest {
+
+    func test_safeFileContext() throws {
+        let tempURL = createTempFolder()
+        let sut = SafeFileContext(fileURL: tempURL)
+
+        sut.acquireDirectoryLock()
+        print("locked")
+        sut.releaseDirectoryLock()
+        print("unlocked")
+
+    }
```

Comment:
what makes this test successful? not timing out?

Reply:
removed as there's nothing to assert